### PR TITLE
Add a configurable space character to LaTeX and HTML engines

### PR DESCRIPTION
**Pull request by ohad**
This is useful in LaTeX since stretchable spaces can muck up the
spacing depending on the configuration

### CODE REVIEW

**Comment by ohad** [comment]
I'm not sure why the run is failing.
The difference seems to be some blank lines are missing from the new output:
[Expected](https://github.com/idris-community/katla/runs/4161949072?check_suite_focus=true#step:10:23)
[Given](https://github.com/idris-community/katla/runs/4161949072?check_suite_focus=true#step:10:119)

(I'm assuming the `make` errors interleaved in the `Given` output is a race condition I should ignore.)

On my machine, both files are identical.
Also, the [source-code](https://github.com/idris-community/katla/pull/17/files#diff-e54da4aecae6afac470898bbf562f4c3dad75d335cf51ea9004514addc730875L98) does seem to include those newlines, so I'm very confused.

File: src/Katla/Batch.idr [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module Katla.Batch
```
Comment (gallais) [inline src/Katla/Batch.idr:1]: I don't think this was meant to be included

File: src/Katla/HTML.idr [hunk 2]
```diff
@@ -24,7 +24,7 @@ annotate (Just dec) s = apply (convert dec) s
     convert Function   = "IdrisFunction"
     convert Data       = "IdrisData"
     convert Keyword    = "IdrisKeyword"
-    convert Bound      = "IdrisBound"
+    convert Bound      = "IdrisBoundxb"
```
[the remark this answers is not in the document]
Comment (ohad) [reply src/Katla/HTML.idr:27]: I guess we should have tests for the html backend

File: tests/examples/raw-snippet/katla-preamble.tex [hunk 3]
```diff
@@ -32,7 +32,7 @@
       \IdrisHlightStyleHole\IdrisHlightFont%
       #1}}}
 
-\newcommand{\RawIdrisHighlight}[3]{{\textcolor{#1}{#2\IdrisHlightFont#3}}}
+\newcommand{\RawIdrisHighlight}[3]{{\textcolor{#1}{\IdrisHlightFont#2{#3}}}}
```
Comment (gallais) [inline tests/examples/raw-snippet/katla-preamble.tex:35]: Is this change intentional?
